src/main/steps/common/task-list/controllers/CaseDetailsGetController.ts

```diff
@@ -0,0 +1,31 @@
+import autobind from 'autobind-decorator';
+import { Response } from 'express';
+
+import { CosApiClient } from '../../../../app/case/CosApiClient';
+import { AppRequest } from '../../../../app/controller/AppRequest';
+import { getCasePartyType } from '../../../prl-cases/dashboard/utils';
+import { DASHBOARD_URL, PARTY_TASKLIST, SIGN_IN_URL } from '../../../urls';
+import { applyParms } from '../../url-parser';
+
+@autobind
+export default class CaseDetailsGetController {
```

Comment:
What's the purpose of this controller? we already have a get case controller to fetch case data

Reply:
It's is retrieve case details & redirect to applicant/respondent task lists.